internal/provider/auth_oidc_test.go

```diff
@@ -231,25 +230,6 @@ func TestAuthLoginOIDC_Login(t *testing.T) {
 	}
 
 	tests := []authLoginTest{
-		{
```

Comment:
The Auth OIDC uses the JWT CLI Auth handler and not the common `AuthLoginCommon` method which performs the token validation. This test is unrelated for the Auth OIDC Login flow, hence removed